module/VuFindTheme/src/VuFindTheme/Initializer.php

```diff
@@ -191,6 +198,32 @@ public function init()
         }
     }
 
+    /**
+     * Get a map of theme aliases to theme names.
+     *
+     * @return array
+     */
+    protected function getThemeAliasMap(): array
+    {
+        if ($this->themeMap === null) {
+            // Set up special-case 'standard' and 'mobile' aliases:
+            $this->themeMap = ['standard' => $this->config->theme];
+            if ($this->mobile->enabled() && isset($this->config->mobile_theme)) {
```

Comment:
I think it's a good idea to get rid of the enabled method, as it's more confusing than helpful. However, this is not the place to do that, since this is a bug fix for a patch release. For now, I've just removed the unnecessary redundant isset check.